src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -600,6 +603,16 @@ def launch_fluent(
 
     del start_container
 
+    # Check if user set PYFLUENT_ADDITIONAL_ARGUMENTS var
+    if os.getenv("PYFLUENT_ADDITIONAL_ARGUMENTS") is not None:
+        # Now check if user used the additional_arguments option
+        if additional_arguments is not None:
+            logger.warning(
+                "Both the 'PYFLUENT_ADDITIONAL_ARGUMENTS' variable and 'additional_arguments' "
+                "options cannot be used together. Using the 'PYFLUENT_ADDITIONAL_ARGUMENTS' option."
+            )
+        additional_arguments = os.environ.get("PYFLUENT_ADDITIONAL_ARGUMENTS")
```

Comment:
I don't agree with this behaviour.  The arguments I explicitly pass to the launch_fluent call should always take precedence over the configured environment.